css/style.css

```diff
@@ -972,9 +990,15 @@ button.button-inline:hover {
 		display: none;
 	}
 
+	#innerBoard {
+		display: flex;
+	}
+
 	.stack {
 		width: 320px;
+		min-width: 320px;
```

Comment:
this doesn't didn't you mention resizing from 200px? Left like this, it has 0 effect because of `width: 320px`